<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Human Variome Project Australian Node &middot; A Jekyll theme
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/hyde.css">
  <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700|Abril+Fatface">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-144-precomposed.png">
                                 <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body class="theme-base-0d">

    <div class="sidebar">
  <div class="container sidebar-sticky">
    <div class="sidebar-about">
       <img src="/img/hvpa_logo.png/" id="logo" />
      <h2>
        <a href="/">
          Human Variome Project Australian Node
        </a>
      </h2>
      <p class="lead">Helping Australian diagnostic laboratories deliver better results for patients through secure data sharing.</p>
    </div>

    <nav class="sidebar-nav">
      <a class="sidebar-nav-item" href="/">About</a>

      

      
      
        
          
        
      
        
          
            <a class="sidebar-nav-item" href="/Who/">Who</a>
          
        
      
        
      
        
          
        
      
        
          
            <a class="sidebar-nav-item" href="/contact/">Contact</a>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
            <a class="sidebar-nav-item" href="/data/data/">Data</a>
          
        
      
        
          
        
      
        
          
        
      
        
          
        
      
        
          
            <a class="sidebar-nav-item" href="/the-data/privacy-ethics/">Privacy & Ethics</a>
          
        
      
        
          
            <a class="sidebar-nav-item" href="/roadmap/">Roadmap</a>
          
        
      
        
          
        
      
        
          
        
      
        
          
            <a class="sidebar-nav-item" href="/the-system/the-system/">The System</a>
          
        
      
    <a href="http://portal.hvpaustralia.org.au/"> HVP Australia Portal </a>
    </nav>

    <p>&copy; 2016. All rights reserved.</p>
  </div>
</div>


    <div class="content container">
      <p>The Human Variome Project Australian Node</p>

<p>The Human Variome Project Australian Node is a national data sharing facility for improving clinical genetic testing services and supporting medical research.</p>

<p>The Node consists of a secure, digital data repository and a suite of collection tools.  It is operated by the University of Melbourne Department of Pathology with the participation of prominent members of the Australian genetic diagnostics and research community. The Australian Nodes has been developed since 2010 with support from Federal Government through the National eResearch Architecture Taskforce (NeAT) and National eResearch Collaboration Tools and Resources (NeCTAR) Projects.</p>

<p>As a Country Node member of the Human Variome Project, the Australian Node is Australia’s contribution to global data collection and sharing efforts.</p>

<p>How it works
Information on genetic test results and interpretations is automatically collected from participating laboratories in Australia at the same time as a report is issued to the medical practitioner requesting the test. All information that can potentially be used to identify an individual is removed from the record before it is securely transferred to the Node repository.</p>

<p>The Node data is made available to registered users in Australia through the Portal where it can be used to aid the diagnosis and treatment of Australian patients.</p>

<p>Some de-identified information on individuals who have consented to research use of their data may be made available to scientists who are undertaking specific research projects into the causes and potential treatments of genetic diseases. Requests for access for research use must undergo review by a Human Research Ethics Committee before being referred to the Australian Node Data Access Committee.</p>

<p>History
The Human Variome Project Australian Node was established by consensus following a meeting of a consortium of Australian clinicians and pathologists in November of 2008.</p>

<p>The Genomic Disorders Research Centre at the Florey Neuroscience Institutes was asked to undertake a pilot study that would examine how an HVP Country Node could be developed for Australia. The pilot phase was funded as a NeAT project supported by the NCRIS Platforms for Collaboration, with funding beginning July 2009.</p>

<p>The pilot phase established and tested software systems for the collection of molecular information from Australian laboratories and defined an ethical framework and management structure under which the Project would continue to operate beyond the lifetime of the NeAT funding. The Project’s pilot phase completed at the end of June 2011, after which the Node will moved to a permanent home within the Department of Pathology at the University of Melbourne.</p>

<p>In September of 2012, the Australian Node Project announced a funding arrangement with the NeCTAR Project to significantly increase the coverage of collection, both in terms of labs and genes, as well as investigate clinical data linkage.</p>

<p>The next phase of development of the HVPA Node will see it able to link with clinical data and to collect and share next-generation sequencing data generated in both clinical and research contexts. We are working out ways in which this genomic –scale data can be safely and securely shared between clinical services to leverage maximum patient benefit from diagnostic genetic information</p>

    </div>

  </body>
</html>